blockchain/My-Wallet-V3-iOS
# fix(KYC): IOS-1272 showing loading view when waiting for Onfido applicantId submission to complete

## Objective

Show loading indicator when waiting for `OnfidoService.submitVerification` to complete.

## Description

Previously, upon submitting identity docs, the user is left to wait for `OnfidoService.submitVerification` to complete which could take a couple of seconds without any visual indicator. With this PR, we are using the `LoadingViewPresenter` to show a loading screen and subsequently hiding it when it completes.

Additionally, this PR includes error handling when submission fails.

## How to Test

Go through KYC, complete Onfido steps, and verify that once you send a video selfie, you see a loading indication while the client sends the Onfido applicantId to Nabu.

## Merge Checklist
